packages/framework/src/Form/FileUploadType.php

```diff
@@ -97,6 +108,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('file', FileType::class, [
                 'multiple' => $options['multiple'],
+                'info_text' => $options['info_text'],
```

Comment:
Why do you pass the option into FileType that doesn't need it? The info_text is displayed inside of this form type.